x-pack/plugins/monitoring/server/routes/api/v1/setup/cluster_setup_status.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import Joi from 'joi';
+import { verifyMonitoringAuth } from '../../../../lib/elasticsearch/verify_monitoring_auth';
+import { handleError } from '../../../../lib/errors';
+import { getCollectionStatus } from '../../../../lib/setup/collection';
+import { getIndexPatterns } from '../../../../lib/cluster/get_index_patterns';
+
+export function clusterSetupStatusRoute(server) {
+  /*
+   * Monitoring Home
+   * Route Init (for checking license and compatibility for multi-cluster monitoring
+   */
+  server.route({
+    method: 'POST',
+    path: '/api/monitoring/v1/setup/collection/{clusterUuid}',
+    config: {
+      validate: {
+        params: Joi.object({
+          clusterUuid: Joi.string().required()
+        }),
+        payload: Joi.object({
+          ccs: Joi.string().optional()
```

Comment:
Any reason the other (non-cluster-specific) API route introduced in this PR doesn't take the this payload?

Reply:
Oversight. Will clean up